src/wallet/wallet2.cpp

```diff
@@ -12126,7 +12126,7 @@ size_t wallet2::import_outputs(const std::pair<size_t, std::vector<tools::wallet
   PERF_TIMER(import_outputs);
 
   THROW_WALLET_EXCEPTION_IF(outputs.first > m_transfers.size(), error::wallet_internal_error,
-      "Imported outputs omit more outputs that we know of");
+      "Consider using `export_outputs all <filename>` when exporting outputs.");
```

Comment:
Is this the correct guidance? `export_outputs all` suggests a higher number of outputs will get exported and the existing error is already saying it has too many.